library/spdm_responder_conformance_test_lib/spdm_responder_test_12_heartbeat_ack.c

```diff
@@ -450,7 +450,7 @@ void spdm_test_case_heartbeat_ack_unexpected_request (void *test_context)
     }
     common_test_record_test_assertion (
         SPDM_RESPONDER_TEST_GROUP_HEARTBEAT_ACK,
-        SPDM_RESPONDER_TEST_CASE_HEARTBEAT_ACK_UNEXPECTED_REQUEST_IN_DHE_SESSION_HS, 1,
+        SPDM_RESPONDER_TEST_CASE_HEARTBEAT_ACK_UNEXPECTED_REQUEST_IN_DHE_SESSION_HS, 3,
```

Comment:
This value 1 is correct, according to https://github.com/DMTF/SPDM-Responder-Validator/blob/main/doc/12.HeartbeatAck.md